packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -228,6 +242,54 @@ export const LiveChromeLarge = (props: ThemeMuxTemplateProps) => html`
   </media-control-bar>
 `;
 
+// prettier-ignore
+export const DvrChromeExtraSmall = VodChromeExtraSmall;
+
+// prettier-ignore
+export const DvrChromeSmall = (props: ThemeMuxTemplateProps) => html`
+  <media-control-bar slot="top-chrome" style="justify-content: flex-end;">
+    ${props.hasCaptions ? MediaCaptionsButton(props) : html``}
+    ${MediaAirplayButton()}
+    ${MediaPipButton()}
+  </media-control-bar>
+  <div slot="centered-chrome" class="mxp-center-controls">
+    ${MediaSeekBackwardButton(props)}
+    ${MediaPlayButton()}
+    ${MediaSeekForwardButton(props)}
+  </div>
+  <media-time-range></media-time-range>
+  <media-control-bar>
+    ${MediaMuteButton()}
+    <media-volume-range></media-volume-range>
+    <slot name="seek-to-live-button"></slot>
+    <div class="mxp-spacer"></div>
+    ${MediaFullscreenButton()}
+    <div class="mxp-padding-2"></div>
+  </media-control-bar>
+`;
+
+// prettier-ignore
+export const DvrChromeLarge = (props: ThemeMuxTemplateProps) => html`
+  <div slot="centered-chrome" class="mxp-center-controls">
+    ${MediaPlayButton()}
+  </div>
+  <media-time-range></media-time-range>
+  <media-control-bar>
+    ${MediaPlayButton()}
+    ${MediaSeekBackwardButton(props)}
+    ${MediaSeekForwardButton(props)}
+    ${MediaMuteButton()}
+    <media-volume-range></media-volume-range>
+    <slot name="seek-to-live-button"></slot>
```

Comment:
is this the main difference between the Vod and Dvr templates?

Reply:
Per out of band convo: for now, yes, though this will change as we implement "fancier" DVR UIs.